pom.xml

```diff
@@ -48,6 +48,7 @@
 		<github.global.server>github</github.global.server>
 		<slf4j.version>1.7.36</slf4j.version>
 		<jedis.module.name>redis.clients.jedis</jedis.module.name>
+		<resilience4j.version>1.7.1</resilience4j.version>
```

Comment:
Can we use Resilience4j 2.0?

Reply:
It would introduce breaking changes. IIRC there was a Decorator object that was removed even though its the most efficient way that I found to get failover to work for our use-case. So I decided to stick with 1.7 until they added a capability that worked around this deprecated way to configure the supplier on every call.